firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -158,15 +162,17 @@ public InstallationResponse createFirebaseInstallation(
         writeFIDCreateRequestBodyToOutputStream(httpURLConnection, fid, appId);
 
         int httpResponseCode = httpURLConnection.getResponseCode();
+        requestLimiter.setNextRequestTime(httpResponseCode);
```

Comment:
What happens in case of exceptions?
Do we need to set a `nextRequestTime`?
I assume not, just checking if you considered that case.

Reply:
Correct, we should & do not set nextRequestTime on exceptions. Instead retry once before throwing unavailable  exception.